### PR TITLE
Sample Json added dummy component.

### PR DESCRIPTION
**Pull request by sarajaur**
# Related Issue

A Pull Request should close a **single** issue; multiple issues can be closed when the issues are **small and related**, but that should be an exception not the rule. Please keep Pull Requests small and targeted; large 'code drops' with dozens of files will be closed and asked to split into reviewable pieces. Reviews that need to be large due to dependencies will be
reviewed on a case-by-case basis.

Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
the issue fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one to aid
in issue tracking.

**NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
page above.

# Description

For all Pull Requests, please describe how the issue was fixed or how the feature was implemented from a summary level. This information will be used to help provide context to the reviewers of the pull request and should be additive to the issue being closed.

     • Created sample json and Qml file with a dummy placeholder component and linked the qml file with 
        "filteredChoiceSet" component. 

     • Jira : SPARK-522904
     
Screenshot 2024-05-20 211333

# Sample Card

If appropriate, please include a link to a card in one of the samples directories that can be used to validate this change. This can be an existing card or a card added with this PR.

# How Verified

How you verified the fix, including one or all of the following:
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

File: source/qml/Samples/QmlVisualizer/qml.qrc [hunk 1]
```diff
@@ -38,5 +38,6 @@
         <file alias="WCustomFocusItem.qml">AdaptiveCardQml/WCustomFocusItem.qml</file>
         <file alias="InputFieldClearIcon.qml">AdaptiveCardQml/InputFieldClearIcon.qml</file>
         <file alias="DrawUtils.js">AdaptiveCardQml/Utils/DrawUtils.js</file>
+		<file alias="FilteredChoiceSetRender.qml">AdaptiveCardQml/FilteredChoiceSetRender.qml</file>
```
Comment (snbehar) [inline source/qml/Samples/QmlVisualizer/qml.qrc:41]: Tab looks weird here. Otherwise LGTM

**Review by snbehar — APPROVED** [review]

**Review by mukuagar — APPROVED** [review]

File: source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml [hunk 2]
```diff
@@ -0,0 +1,276 @@
+import "AdaptiveCardUtils.js" as AdaptiveCardUtils
+import AdaptiveCards 1.0
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.3
+
+
+ComboBox {
```
Comment (ramkrishnajana) [inline source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml:8]: As per this PR, we are just showing a placeholder. Is the code in this file valid?

**Review by ramkrishnajana — CHANGES_REQUESTED** [review]

File: source/qml/Library/RendererQml/ChoiceSetRender.cpp [hunk 3]
```diff
@@ -102,7 +106,8 @@ void ChoiceSetElement::renderChoiceSet(RendererQml::ChoiceSet choiceSet, Rendere
         {
             choice_Text = choice.text;
             choice_Value = choice.value;
-            model << "{ valueOn: String.raw`" << RendererQml::Utils::getBackQuoteEscapedString(choice_Value) << "`, text: String.raw`" << RendererQml::Utils::getBackQuoteEscapedString(choice_Text) << "`},\n";
+            model << "{ valueOn: String.raw`" << RendererQml::Utils::getBackQuoteEscapedString(choice_Value)
+                  << "`, text: String.raw`" << RendererQml::Utils::getBackQuoteEscapedString(choice_Text) << "`},\n";
```
[the remark this answers is not in the document]
Comment (sarajaur) [reply source/qml/Library/RendererQml/ChoiceSetRender.cpp:110]: Indentation changes.

File: source/qml/Library/RendererQml/ChoiceSetRender.cpp [hunk 4]
```diff
@@ -111,15 +116,37 @@ void ChoiceSetElement::renderChoiceSet(RendererQml::ChoiceSet choiceSet, Rendere
...
+                             choiceSet.choices.end(),
+                             [target](const RendererQml::Checkbox& options) { return options.value == target; }) -
+                         choiceSet.choices.begin();
+
+            if (index < (signed int)(choiceSet.choices.size()))
+            {
+                mChoiceSetColElement->Property("_comboboxCurrentIndex", std::to_string(index));
+            }
+        }
+    }
```
[the remark this answers is not in the document]
Comment (sarajaur) [reply source/qml/Library/RendererQml/ChoiceSetRender.cpp:130]: just indentation change.

File: source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml (hunk 2 above)
Comment (sarajaur) [reply source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml:8]: Yes Ram ,It's valid.

File: source/qml/Library/RendererQml/ChoiceSetRender.cpp [hunk 5]
```diff
@@ -111,15 +116,37 @@ void ChoiceSetElement::renderChoiceSet(RendererQml::ChoiceSet choiceSet, Rendere
...
+
+        model << "[";
+        for (const auto& choice : choiceSet.choices)
+        {
+            choice_Text = choice.text;
+            choice_Value = choice.value;
+            model << "{ valueOn: String.raw`" << RendererQml::Utils::getBackQuoteEscapedString(choice_Value)
+                  << "`, text: String.raw`" << RendererQml::Utils::getBackQuoteEscapedString(choice_Text) << "`},\n";
+        }
+        model << "]";
```
[the remark this answers is not in the document]
Comment (sarajaur) [reply source/qml/Library/RendererQml/ChoiceSetRender.cpp:148]: I tried optimizing but I didn't get any other solution.

**Review by ramkrishnajana — APPROVED** [review]

**Pull request merged**